contracts/FlowIDTableStaking.cdc

```diff
@@ -79,15 +79,15 @@ pub contract FlowIDTableStaking {
     /// value = decimal number between 0 and 1 indicating a percentage
     access(contract) var rewardRatios: {UInt8: UFix64}
 
-    /// The percentage of rewards that every node operator takes from 
+    /// The percentage of rewards that every node operator takes from
     /// the users that are delegating to it
     access(contract) var nodeDelegatingRewardCut: UFix64
 
     /// Paths for storing staking resources
-    pub let NodeStakerStoragePath: Path
```

Comment:
@turbolent Is changing the type of the path here something that is going to be compatible with the way that contracts are upgraded? (cannot add or remove state)
Should I make a test to see if doing an upgrade like this works properly?

Reply:
Yes, this is just a change of the type of an existing field, which is supported. This doesn't add or remove any state.

Reply:
Of course, the value stored in the existing field needs to be compatible with the new, more specific type, which I checked for each field.